docusaurus/docs/dev-docs/api/rest/populate-select.md

```diff
@@ -251,12 +251,16 @@ await request(`/api/articles?${query}`);
 ```js
 // Object method
 const qs = require('qs');
-const query = qs.stringify({
```

Comment:
this results into

<img width="485" alt="Screenshot 2023-03-25 at 11 31 16" src="https://user-images.githubusercontent.com/1225343/227711852-66fefd0a-55a3-471e-8f93-f96c80c27639.png">